components/elm/src/cpl/lnd_import_export.F90

```diff
@@ -1145,7 +1145,7 @@ subroutine lnd_import( bounds, x2l, atm2lnd_vars, glc2lnd_vars)
        if (co2_type_idx == 0) then                    ! CO2 constant, value from namelist
          co2_ppmv_val = co2_ppmv
        else if (co2_type_idx == 1) then               ! CO2 prognostic, value from coupler field
-         co2_ppmv_val = x2l(index_x2l_Sa_co2prog,i)
+         co2_ppmv_val = 0d0 !x2l(index_x2l_Sa_co2prog,i)
```

Comment:
Why is the coupler field for prognostic CO2 replaced by 0?

Reply:
I have changed this back.